Moli/Core/Screens/Home/Live/MoliRoomItemView.swift

```diff
@@ -0,0 +1,111 @@
+import SwiftUI
+
+struct MoliRoomItemView: View {
+    var room: API.Room.HomeRoomResponse
+    
+    @State private var hosts: [API.Room.HostInfo] = []
+    @State private var viewerProfiles: [API.User.UserMinified] = []
+    @State private var extraViewersCount = 0
+    
+    private func setHostProfiles() {
+        self.hosts = room.hostsInfo
+    }
+    
+    private func setViewerProfiles() {
+        let viewerProfiles = Array(room.previewViewers.prefix(4))
+        
+        self.viewerProfiles = API.User.UserMinified.convertFrom(viewerProfiles)
+        self.extraViewersCount = self.room.viewers.count - viewerProfiles.count
+    }
+    
+    var body: some View {
+        ZStack(alignment: .topLeading) {
+            if let imageURL = room.thumbnail?.thumbnailUrl, let url = URL(string: imageURL) {
+                AsyncImage(url: url) { image in
+                    image.resizable()
+                        .ignoresSafeArea()
+                } placeholder: {
+                    Image("moliRoomsFallbackBG")
+                        .resizable()
+                        .ignoresSafeArea()
+                }
+            }
+            VStack(alignment: .leading) {
+                HStack {
+                    Text(room.title.replacingOccurrences(of: " ", with: "\n").uppercased())
+                        .font(Font.headingLarge.weight(.bold))
+                        .multilineTextAlignment(.leading)
+                        .padding(8)
+                        .background(room.title == "GRWM" ? Color.newYellow2 : Color.newPurple)
```

Comment:
:art: Här om dagen lärde mig Denhi om computed values, något jag har börjat att använda i rummet när jag kan. Det här kodsnuttet passar perfekt för ett sånt scenario, och det är helt enkelt att du kan skapa en private variabel som beror på/läser av `room.title`, så att:
```swift
private var titleBackgroundColor: Color { room.title == "GRWM" ? Color.newYellow2 : Color.newPurple } 
```

Då kan du lägga in den:

```suggestion
                        .background(titleBackgroundColor)
```

Du skulle kunna göra samma sak med setHostProfiles och setViewerProfiles om du vill 👍🏼